signal_interface/src/http/handlers/encrypt.cpp

```diff
@@ -34,12 +34,12 @@ int postEncryptKey(struct mg_connection *conn, void *cbdata, char *dbPath, char*
 
   if (!cJSON_IsString(recipientId) || !cJSON_IsNumber(deviceId)) {
     spdlog::error("[{0}] Missing Params", endpointId, bufferData);
-    mg_send_http_error(conn, 400, "%s", "Missing params");
+    sendError(conn, 400, "Missing Params");
```

Comment:
Code 400 is used for multiple purposes, define just one, either "Request Data to Big" or "Missing Params", but not both.